Singularity

```diff
@@ -0,0 +1,13 @@
+From:nfcore/base
+Bootstrap:docker
+
+%labels
+    MAINTAINER Andreas Wilm <wilma@gis.a-star.edu.sg>
+    DESCRIPTION Singularity image containing all requirements for the nf-core/vipr pipeline
+
+%files
+    environment.yml /
+
+%post
+    /opt/conda/bin/conda env update -n root -f /environment.yml
+/opt/conda/bin/conda clean -a
```

Comment:
Indentation alert 😱

indentation police 🚓 👮

Reply:
I just noticed that I forgot some stuff, so I'll make other PRs to fix all that